rustfmt.toml

```diff
@@ -1,2 +0,0 @@
-# https://github.com/rust-lang/rustfmt/issues/4991
-imports_granularity = "Module"
```

Comment:
Nightly `cargofmt` is failing dprint check due to printing to `stderr` strings like `rewriting static`. It does not seem to introduce any diff in formatting but dprint github action seems to get confused by those logs.

I adjusted the action to use stable `cargofmt` and it seems to produce no diff either. So maybe we can switch to stable version? @mgeisler

Reply:
Yeah, this is fine for now! The reason we don't see any diff is that the default is `Preserve`.